atomic_scanners/worker_start_test.py

```diff
@@ -1,213 +0,0 @@
-
-    logger.log(level=logging.INFO,
-               msg="Executing atomic scan: %s" % cmd)
-
-    process = subprocess.Popen([
-        'atomic',
-        'scan',
-        "--scanner=pipeline-scanner",
-        "--rootfs=%s" % image_rootfs_path,
-        "%s" % image_id
-    ], stderr=subprocess.PIPE, stdout=subprocess.PIPE)
-
-    out, err = process.communicate()
-
-    # logger.log(level=logging.INFO,
-    #            msg="Created container with ID: %s" % container.get('Id'))
-
-    # conn.start(container=container.get('Id'))
-
-    # logger.log(level=logging.INFO,
-    #            msg="Started container with ID: %s" % container.get('Id'))
-
-    # time.sleep(10)
-    # logs = conn.logs(container=container.get('Id'))
-
-    # logger.log(level=logging.INFO, msg="Stopping test container")
-    # conn.stop(container=container.get('Id'))
-
-    # logger.log(level=logging.INFO, msg="Removing the test container")
-    # conn.remove_container(container=container.get('Id'), force=True)
-
-    if out != "":
-        # TODO: hacky and ugly. figure a better way
-        output_json_file = os.path.join(
-            out.strip().split()[-1].split('.')[0],
-            "_%s" % image_rootfs_path.split('/')[1],
-            "image_scan_results.json"
-        )
-
-        if os.path.exists(output_json_file):
-            json_data = json.loads(open(output_json_file).read())
-        else:
-            logger.log(level=logging.FATAL,
-                       msg="No scan results found at %s" % output_json_file)
-            raise
-    else:
-        logs = ""
-
-    logger.log(level=logging.INFO,
-               msg="Unmounting image's rootfs from %s" % image_rootfs_path)
-
-    mount_object.unmount()
-
-    os.rmdir(image_rootfs_path)
-
-    logger.log(level=logging.INFO, msg="Removing the image %s" % image_full_name)
-    conn.remove_image(image=image_full_name, force=True)
-
-    logger.log(level=logging.INFO, msg="Finished test...")
-
-    # if msg != "" and logs != "":
-    if json_data != None:
-        d = {
-            "image": image_full_name,
-            "msg": "Container image requires update",
-            "logs": json.dumps(json_data),
-            "action": "complete_schedule_scan",
-            "name_space": namespace
-        }
-    else:
-        d = {
-            "image": image_full_name,
-            "msg": "No updates required",
-            "logs": "",
-            "action": "complete_schedule_scan",
-            "name_space": namespace
-        }
-    bs.use("master_tube")
-    jid = bs.put(json.dumps(d))
-    logger.log(
-        level=logging.INFO,
-        msg="Put job on master tube with id: %d" % jid
-    )
-
-bs = beanstalkc.Connection(host=BEANSTALKD_HOST)
-bs.watch("start_schedule_scan")
-
-while True:
-    try:
-        job = bs.reserve()
-        job_data = json.loads(job.body)
-        test_job_data(job_data)
-        job.delete()
-    except Exception as e:
-        logger.log(level=logging.FATAL, msg=e.message)
```

Comment:
This file was moved into `beanstalk_worker`  directory and it needed to be removed.